K2Bridge/Visitors/KustoQLOperators.cs

```diff
@@ -34,6 +34,8 @@ public static class KustoQLOperators
         public const string Min = "min";
         public const string Max = "max";
         public const string Sum = "sum";
+        public const string Percentile = "percentile";
```

Comment:
I think this one is redundant and not used anywhere

Reply:
removed